src/common/components/home/MarketplacePreview.tsx

```diff
@@ -0,0 +1,137 @@
+
+const LoadingPreview = () => {
+  return (
+    <Container>
+      <div className="h-full w-full animate-pulse bg-gray-700" />
+    </Container>
+  );
+};
+
+interface MarketplacePreviewProps {
+  loading: boolean;
+  data?: MarketplacePreviewData;
+}
+
+const MarketplacePreview: FC<MarketplacePreviewProps> = ({ loading, data }) => {
+  const [showDetails, setShowDetails] = useState(false);
+  const { track } = useAnalytics();
+
+  useEffect(() => setShowDetails(isTouchScreenOnly()), []);
+
+  const onMouseEnter = useCallback(() => setShowDetails(true), []);
+  const onMouseLeave = useCallback(() => setShowDetails(isTouchScreenOnly()), []);
+  const onClickMarketplaceLink = useCallback(() => {
+    track('Marketplace Selected', {
+      event_category: 'Discovery',
+      event_label: data ? data.subdomain : "unknown-subdomain",
+    });
+  }, [data, track]);
+
+  if (loading || !dataAreComplete(data)) {
+    return <LoadingPreview />;
+  }
+
+  // all required data are available after checking dataAreComplete()
+  data = data!;
+
+  const marketplaceUrl: string = `https://${data.subdomain}.holaplex.market`;
+  const nftVolumeStr: string = Number.parseInt(data.stats.nfts).toLocaleString();
+  const priceSol: number = Number.parseFloat(data.auctionHouse.stats.floor) / LAMPORTS_PER_SOL;
+
+  return (
+    <Container onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
+      {/* preview image */}
+      <div className="relative h-full w-full">
+        <a href={marketplaceUrl} target="_blank" rel="noreferrer" onClick={onClickMarketplaceLink}>
+          <img
+            src={imgOpt(data.bannerUrl, 600)}
+            alt={`${data.name}`}
+            className="absolute w-full object-cover"
+          />
+        </a>
+
+        {/* preview gradient overlay */}
+        <div className="pointer-events-none absolute h-full w-full bg-gradient-to-b from-black/20 to-black/70" />
+      </div>
+
+      {/* creator icons
+            allow pointer events through the container div for clickable preview image while also allowing
+            pointer events on the creator icons */}
+      <div className="pointer-events-none absolute top-0 left-0 h-full w-full select-none pl-5 pt-5">
+        <div className="pointer-events-auto">
+          <AvatarIcons
+            creators={
+              data.creators.map((c) => {
+                return { address: c.creatorAddress };
+              }) || []
+            }
+          />
+        </div>
+      </div>
+
+      {/* marketplace name, NFT volume, and floor price section */}
+      <div className="pointer-events-none absolute bottom-0 left-0 flex w-full flex-col p-5">
+        <span className="text-xl font-semibold text-white">{data.name}</span>
+
+        {/* NFT volume and floor price row container
+                Using height and opacity (rather than 'display') to animate bottom-text appearing */}
+        <div
+          className={`${
+            showDetails ? 'h-8 opacity-100' : 'h-0 opacity-0'
```

Comment:
I think you can get rid off this state by using the tailwind hover: selector https://tailwindcss.com/docs/hover-focus-and-other-states#hover-focus-and-active

Reply:
The issue is that touchscreen-only devices cant hover, and @bellebellesol wants those details to show in that case. That's why I added that utility method for checking touchscreen-only, and set the appearance accordingly.